components/src/web-components/visualization/gs-relative-growth-advantage.tsx

```diff
@@ -63,15 +63,24 @@ export class RelativeGrowthAdvantageComponent extends PreactLitAdapter {
     headline: string | undefined = 'Relative growth advantage';
 
     /**
-     * The size of the component.
+     * The width of the component.
      *
-     * If not set, the component will take the full width of its container with height 700px.
+     * If not set, the component will take the full width of its container.
      *
-     * The width and height should be a string with a unit in css style, e.g. '100%', '500px' or '50vh'.
-     * If the unit is %, the size will be relative to the container of the component.
+     * Visit https://genspectrum.github.io/dashboards/?path=/docs/components-size-of-components--docs for more information.
      */
-    @property({ type: Object })
-    size: { width?: string; height?: string } | undefined = undefined;
+    @property({ type: String })
+    width: string | undefined = undefined;
+
+    /**
+     * The height of the component.
+     *
+     * If not set, the component will use the default height of 700px.
```

Comment:
```suggestion
     * If not set, the component will use the default height of 600px.
```
At least that's what the Preact component has: `const defaultSize = { height: '600px', width: '100%' };`